website/docs/source/docs/boxes/base_macos.html.md

```diff
@@ -0,0 +1,160 @@
+---
+page_title: "Creating a macOS Base Box Manually"
+sidebar_current: "boxes-macos-base"
+---
+
+# Creating a macOS Base Box Manually
+
+Prior to reading this page, please check out the [basics of the Vagrant
+box file format](https://www.vagrantup.com/docs/boxes/format.html).
+
+## Pre-requisites
+
+* Install Vagrant, you can find the instructions [here](/vagrant-parallels/docs/installation)
+
+* Start Parallels Desktop and create a new macOS virtual machine as outlined  in [KB 125561](http://kb.parallels.com/125561/).
+    <div class="alert alert-info">
+    <p>
+        While creating the VM, give the username as ‘vagrant’ only.<br />
+        If this is a public box you should set the password to ‘vagrant’ as well.
+    </p>
+    </div>
+
+  * Enable passwordless sudo for macOS virtual machine
+  * Open Terminal in macOS virtual machine and run:
+
+     ```bash
+     $ sudo visudo
+     ```
+
+    Edit the line:
+
+    ```bash
+    %admin ALL=(ALL) ALL
+    ```
+
+    To Say:
+
+    ```bash
+    %admin ALL=(ALL) NOPASSWD: ALL
+    ```
+
+    Now, you should be able to run sudo without password.
+
+* Install Parallels Tools inside the macOS virtual machine.  
+    macOs 13 > **Install Parallels Tools**
+  ![install_parallels_extension](/images/install_extension.gif)
+
+      <div class="alert alert-info">
+    <p>
+        <strong>Note:</strong> Installing the Parallels Tools will require a reboot
+    </p>
+    </div>
+
+* Boot macOS virtual machine and enable Remote Login (System Settings > General > Sharing > Enable Remote Login). Don't forget to give “Full disk access to users”, do allow “All users” (in the same settings, press ![info](/images/info_32.png) button)
+![Parallels Desktop](/images/allow_sharing.gif)
+* Restart macOS virtual machine for the Remote Login to take effect.
```

Comment:
**suggestion:** Let's use a better wording and remove the line about the second reboot, since it's not necessary:
```suggestion
* Reboot macOS virtual machine and then enable Remote Login (_System Settings > General > Sharing > Enable Remote Login_). Select “All users” and also enable “Full disk access to users”, as shown on the picture below:
![allow_remote_login](/images/allow_remote_login.gif)
```

Reply:
implemented